### ISSUE TITLE
ModuleNotFoundError: No module named 'requests_cache.core'

### ISSUE THREAD

**Issue by bwakkie** [gbif/pygbif#93]
I got the following error after pip install ...
```
Python 3.9.9 (main, Nov 20 2021, 21:30:06)
[GCC 11.1.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from pygbif import occurrences as occ
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/sliss/.local/lib/python3.9/site-packages/pygbif/__init__.py", line 44, in <module>
    from .caching import caching
  File "/home/sliss/.local/lib/python3.9/site-packages/pygbif/caching.py", line 2, in <module>
    from requests_cache.core import remove_expired_responses
ModuleNotFoundError: No module named 'requests_cache.core'
```
Solution: edit pygbif/caching.py
And change in line 2:  requests_cache.core to requests_cache

(source: https://requests-cache.readthedocs.io/en/stable/user_guide/troubleshooting.html 
ModuleNotFoundError: No module named 'requests_cache.core': This module was deprecated in v0.6 and removed in v0.8. Just import from requests_cache instead of requests_cache.core.)

**Comment by elsadg** [comment]
Hi, I am still having issues after the change `requests_cache.core to requests_cache` was made in line 2.

**Comment by bwakkie** [comment]
What type of issues? Can you add an error message?

**Comment by elsadg** [comment]
I was getting exactly the same message you get in your first comment but now it seems to work fine. 

```
    from requests_cache.core import remove_expired_responses
ModuleNotFoundError: No module named 'requests_cache.core'
```

**Comment by agrosh24** [comment]
I am having this exact same problem, I cannot get it to work. I have the same error message as well.

**Comment by bwakkie** [comment]
@agrosh24 Did you try the suggested solution from my first post?

**Comment by Becheler** [comment]
I am running into the same issue.

Installing from pypi with `pip3 install pygbif` leads to
```bash
ModuleNotFoundError: No module named 'requests_cache.core'
```

Installing development version with `pip3 install git+git://github.com/sckott/pygbif.git#egg=pygbif` leads to
```bash
 ModuleNotFoundError: No module named 'appdirs'
```
And after that what seems like a cascade of failed dependencies (beginning with appdirs, geojson_rewind then geomet) :/

**Comment by juanpac96** [comment]
additionally I wanted to complement the previous answer, in case you are using google colab. the way i solved the problem was like this.

```
!pip install folium==0.2.1
!pip3 install appdirs geojson_rewind geomet requests_cache
!pip3 install git+git://github.com/sckott/pygbif.git#egg=pygbif
```

Even so, I think that some libraries should be updated or changed to prevent these incompatibility problems and not have to use so many previous configurations to be able to run the GBIF client.

**Comment by fkyeeb** [comment]
If you go onto https://requests-cache.readthedocs.io/en/stable/user_guide/troubleshooting.html. The request_core package says: "requests_cache.core: This module was deprecated in v0.6 and removed in v0.8. Just import from requests_cache instead of requests_cache.core."

A simple solution is to open the python file in the pygbif package that is importing requests_cache.core, and change that to import requests_cache.

**Comment by MattBlissett** [comment]
Thanks for the comments everyone.

We've just released version 0.6.1, which updates the required dependencies.
